src/control/cmd/daos/filesystem.go

```diff
@@ -262,27 +262,69 @@ func (cmd *fsGetAttrCmd) Execute(_ []string) error {
 	defer cleanup()
 
 	var attrs C.dfs_obj_info_t
-	if err := dfsError(C.fs_dfs_get_attr_hdlr(ap, &attrs)); err != nil {
+	var cmode C.mode_t
+	if err := dfsError(C.fs_dfs_get_attr_hdlr(ap, &attrs, &cmode)); err != nil {
 		return errors.Wrapf(err, "%s failed", fsOpString((ap.fs_op)))
 	}
 
 	var oclassName [16]C.char
 	C.daos_oclass_id2name(attrs.doi_oclass_id, &oclassName[0])
 
+	var diroclassName [16]C.char
+	var fileoclassName [16]C.char
+	if C.mode_is_dir(cmode) {
+		C.daos_oclass_id2name(attrs.doi_dir_oclass_id, &diroclassName[0])
+		C.daos_oclass_id2name(attrs.doi_file_oclass_id, &fileoclassName[0])
+	}
+
 	if cmd.JSONOutputEnabled() {
-		jsonAttrs := &struct {
-			ObjClass  string `json:"oclass"`
-			ChunkSize uint64 `json:"chunk_size"`
-		}{
-			ObjClass:  C.GoString(&oclassName[0]),
-			ChunkSize: uint64(attrs.doi_chunk_size),
+		if C.mode_is_dir(cmode) {
+			jsonAttrs := struct {
+				ObjAttr struct {
+					ObjClass string `json:"oclass"`
+				} `json:"Object"`
```

Comment:
I think in general we've standardised on lower case keys in json haven't we?
```suggestion
				} `json:"object"`
```

Reply:
is that documented somewhere?

Reply:
Not that I'm aware of but I have seen PRs to move things to lower-case and I have seen PRs to update tests after the case was updated and not all test-cases were changed at the time.  A quick git grep of the source shows 78 matches for `json:"` all of which are lower case throughout.

Reply:
but it's not a "standard" if it's not documented or at least gofmt does not throw an error on it.

Reply:
I wouldn't expect gofmt to throw an error on it, to go it's just a string.

https://daosio.atlassian.net/browse/DAOS-10579 is an old ticket but covers making changes so that snake_case rather than CamelCase is used throughout.  I'm not aware of this being writing down in a document as a policy anywhere but it seems clear to me that there was a decision made on this at some point and it seems reasonable to stick to it.

Reply:
i will argue again with your "standardize" comment. sorry but how is it clear that a decision is made on this just because there was a ticket created 2 years ago..
ill push the change, but if there is some standard as you claim for this that we want applied, it has to be clearly documented in the coding rules. otherwise it's just a preference and **not** a standard.

Reply:
Perhaps standardised was too strong a term to use here.  Catching issues like this is part of the review process and documentation/process improvements are always welcome.